ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/behavior/actions/RDXFootPoseAction.java

```diff
@@ -174,7 +166,7 @@ public void process3DViewInput(ImGui3DViewInput input)
       boolean isClickedOn = isMouseHovering && input.mouseReleasedWithoutDrag(ImGuiMouseButton.Left);
       if (isClickedOn)
       {
-         adjustGoalPose.set(true);
+         poseGizmo.setSelected(true);
       }
```

Comment:
Consider when you're looking through the things that happen to poseGizmo. It's useful to see at a glance that is can only ever turn it to true. It will never turn it to false.
![image](https://github.com/user-attachments/assets/780760b3-48f3-4151-9251-8403c66bcf43)